lib/holidays.rb

```diff
@@ -161,6 +161,22 @@ def to_weekday_if_boxing_weekend(date)
       DateCalculatorFactory.weekend_modifier.to_weekday_if_boxing_weekend(date)
     end
 
+    def to_tuesday_if_sunday_or_monday_if_saturday(year)
+      DateCalculatorFactory.weekend_modifier.to_tuesday_if_sunday_or_monday_if_saturday(year)
+    end
+
+    def to_monday_if_weekend(date)
```

Comment:
This already exists on L156 above, we don't need to add it again. Please remove it.